python/ray/util/sgd/torch/batch_logs_reporter.py

```diff
@@ -0,0 +1,29 @@
+import ray
+
+
+@ray.remote(num_cpus=0)
+class BatchLogsReporter:
```

Comment:
Should this be private? `_Batch..`

Reply:
It's not exported from the main file, but sure --- will do